tests/pytorch_pfn_extras_tests/training_tests/test_extension_entry.py

```diff
@@ -0,0 +1,73 @@
+import tempfile
```

Comment:
* As a *unit* test, please use ExtensionsManager instead of Trainer.
* Please try to minimize the code, I think `MyModel` or `MyModelWithLossFn` is too much to test ExtensionEntry.
* Please make sure that extensions are correctly registered (attributes are overridden as expected) and invoked as intended.